jina/docker/hubapi.py

```diff
@@ -161,6 +161,20 @@ def _make_hub_table(manifests):
     return info_table
 
 
+def _validate_docker_login(logger):
```

Comment:
This logic is not correct. We need to fetch docker credentials using hubapi.

Reply:
using `hubapi` for fetching docker credentials